src/_product-updates/2024-08-10-feedbacks-gathering.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentation feedback 
+tags: [New]
+image: /images/changelog/feedback.png
+---
+
+![feedback.png](/images/changelog/feedback.png)
+
+Feedback from real-life users is the best way to improve your documentation. Our last update got you covered! 
+Say hi to the new "Give feedback" button, designed to stand out from the rest of the topbar.
+
+It can be added through a new vendor extension: `x-feedbackLink`, directly in the `info` object of your API definition file. 
+
+The `x-feedbackLink`is an object composed of two elements: 
```

Comment:
Do we want to be this precise in a Product Update (PU)? I’m wondering if this section wouldn’t be better suited for the Help Center (and then we could redirect in the PU to the appropriate page there). As it stands, isn’t the PU diverging a bit from its intended purpose?

Reply:
I thought it was nice to be able to jump into code directly, as the process is pretty "straight-forward". But I guess we could just say "hey, it's an object composed of an url that can be used for xxx, and a label", without saying more?

Reply:
I reworked it, will be updated with the next commit 🙏